zanata-client-commands/src/main/java/org/zanata/client/commands/FileMappingRuleHandler.java

```diff
@@ -108,9 +116,9 @@ private boolean matchFileExtensionWithProjectType(
      */
     public String getRelativeTransFilePathForSourceDoc(
             QualifiedSrcDocName qualifiedSrcDocName,
-            @Nonnull LocaleMapping localeMapping) {
+            @Nonnull LocaleMapping localeMapping, Optional<String> transExt) {
```

Comment:
Please give `transExt` a more meaningful name. Is it the translation file extension?